Ginger/GingerCoreNET/Run/GingerRemoteExecutionUtils.cs

```diff
@@ -98,13 +98,13 @@ private List<RunSetReport> ConvertResponsInRunsetReport(string result)
                     GUID = runsetHLInfo.ExecutionID.ToString(),
                     Name = runsetHLInfo.RunsetName,
                     Description = "",
-                    StartTimeStamp = Convert.ToDateTime(runsetHLInfo.StartTime).ToUniversalTime(),
-                    EndTimeStamp = Convert.ToDateTime(runsetHLInfo.EndTime).ToUniversalTime(),
+                    StartTimeStamp = Convert.ToDateTime(runsetHLInfo.StartTime).ToLocalTime(),
```

Comment:
Please check if we can keep ToUniversalTime and convert format to match local machine time format